packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -519,9 +533,9 @@ class _DropdownMenuState<T> extends State<DropdownMenu<T>> {
         trailingIcon: entry.trailingIcon,
         onPressed: entry.enabled
           ? () {
-              textEditingController.text = entry.label;
-              textEditingController.selection =
-                TextSelection.collapsed(offset: textEditingController.text.length);
+              _textEditingController.text = entry.label;
```

Comment:
Don't we need to set the `_textEditingController.value` instead?

Per the docs, setting the text and selection in separate statements should be avoided.